src/services/beerService.tsx

```diff
@@ -1,52 +1,54 @@
 import firebase from "../utils/firebaseConfig";
 
+export type Beer = {
+  id: string,
+  name: string, 
+  style: string,
+  brewery: string
+};
+
 const beerService = {
-  findAll: () => {
+  findAll: (): Array<Beer> => {
     const beersDB = firebase.database().ref("beersDB");
-    let list = [];
+    let list: (object)[] = [];
     beersDB.on("value", (snapshot) => {
       let previousList = snapshot.val();
       for (let id in previousList) {
         list.push({ id, ...previousList[id] });
       }
     });
-    return list;
+    return list as Array<Beer>;
   },
 
-  findById: (idParam) => {
+  findById: (idParam: string): any => {
```

Comment:
this returns a `Beer`

Reply:
Until we fix async issues, I think we can just do `findByid: (idParam: string): Beer | undefined`